tests/DevApps/WebAppCallsWebApiCallsGraph/TodoListService/Startup.cs

```diff
@@ -65,6 +65,14 @@ public void ConfigureServices(IServiceCollection services)
                      .AddInMemoryTokenCaches();
 #endif
 
+            services.Configure<JwtBearerOptions>(JwtBearerDefaults.AuthenticationScheme,
+     options =>
+     {
+         options.RequireHttpsMetadata = false;
+     }
+     
```

Comment:
remove line and check spacing

Reply:
Removed as it's not needed. The SimulateOidc runs on https: